invenio_app_ils/ui/src/invenio_app_ils/common/api/loans/serializer.js

```diff
@@ -0,0 +1,42 @@
+import _isEmpty from 'lodash/isEmpty';
+import { fromISO, toShortDateTime } from '../date';
+
+function serializeResponse(hit) {
+  const result = {};
```

Comment:
I would propose in general to do something like `let result = {...hit.metadata}` and then in then serilizer just handle only the fields that need some special check or rename. WDYT?

Reply:
done!